<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QToasterBrowserLCD</class>
 <widget class="QWidget" name="QToasterBrowserLCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>115</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#QToasterBrowserLCD
{
background-color: transparent;
}</string>
  </property>
  <widget class="QLineEdit" name="stompAEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="Line" name="browserLine2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>83</y>
     <width>299</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#browserLine2
{
	color:  rgb(0, 0, 0);
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="rigNameEdit">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>51</y>
     <width>300</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Oxygen-Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#rigNameEdit
{
background-color: transparent;
}
</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>---</string>
   </property>
  </widget>
  <widget class="Line" name="browserLine1">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>53</y>
     <width>299</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#browserLine1
{
	color:  rgb(0, 0, 0);
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="browserLine3">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>54</y>
     <width>1</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#browserLine3
{
	color:  rgb(0, 0, 0);
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="stompBEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stompCEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stompDEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stompModEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="reverbEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>REV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stompXEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delayEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>35</y>
     <width>34</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>DLY</string>
   </property>
  </widget>
  <widget class="Line" name="browserLine4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>42</y>
     <width>152</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#browserLine4
{
	color:  rgb(0, 0, 0);
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="browserLine5">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>42</y>
     <width>152</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#browserLine5
{
	color:  rgb(0, 0, 0);
	background-color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>85</y>
     <width>420</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget {background-color: transparent;}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="knobsPage">
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QLabel" name="bassLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>12</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Bass</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="middleLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>12</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Middle</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="trebleLabel">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>12</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Treble</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="presenceLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>12</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Presence</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="bassValueLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;0.0&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="middleValueLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;0.0&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="trebleValueLabel">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>0</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;0.0&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="presenceValueLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>95</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;0.0&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="scalePage">
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>414</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar#progressBar
{
background-color:transparent;
border-radius: 3px;
}
QProgressBar#progressBar::chunk
{
     background-color: rgb(50,50,50);
}</string>
     </property>
     <property name="value">
      <number>47</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>0</y>
       <width>420</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="empty"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>53</y>
     <width>108</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
border-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QLineEdit" name="rigAuthorEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>108</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rigTypeEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>16</y>
      <width>108</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Guitar Rig</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>35</y>
     <width>112</width>
     <height>15</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLineEdit" name="ampNameEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>112</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
background-color: transparent;
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>---</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="browserViewComboBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>90</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background-color: rgb(50,50,50);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border-radius: 3px;
padding-left:7px;
}
QComboBox::drop-down {width: 0px; border-style: none}
</string>
   </property>
   <property name="currentText">
    <string>All Rigs</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>7</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>All Rigs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Favorites</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Non-Favorites</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>My Rigs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Current Author</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Last Imported</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Just Bass Rigs</string>
    </property>
   </item>
  </widget>
  <zorder>browserLine5</zorder>
  <zorder>browserLine4</zorder>
  <zorder>stompAEdit</zorder>
  <zorder>browserLine2</zorder>
  <zorder>rigNameEdit</zorder>
  <zorder>browserLine1</zorder>
  <zorder>browserLine3</zorder>
  <zorder>stompBEdit</zorder>
  <zorder>stompCEdit</zorder>
  <zorder>stompDEdit</zorder>
  <zorder>stompModEdit</zorder>
  <zorder>reverbEdit</zorder>
  <zorder>stompXEdit</zorder>
  <zorder>delayEdit</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>widget</zorder>
  <zorder>frame</zorder>
  <zorder>browserViewComboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
